### ISSUE TITLE
Error occurred while enabling MassiveCore - 1.16.1 - Spigot

### ISSUE THREAD

**Issue by uwelehmann** [magnusulf/MassiveCore#5]
Server: Spigot build with "BuildTools.jar --rev 1.16.1"
Logs:

[MassiveCore] Enabling MassiveCore v3.3.0[21:53:27] [Server thread/INFO]: [MassiveCore] Enabling MassiveCore v3.3.0
[21:53:27] [Server thread/INFO]: ^[[0;36;22m[^[[0;36;1mMassiveCore 3.3.0^[[0;36;22m] ^[[0;33;1m=== ENABLE START ===^[[m
[21:53:27] [Server thread/INFO]: ^[[0;36;22m[^[[0;36;1mMassiveCore 3.3.0^[[0;36;22m] ^[[0;33;1m^[[0;33;1mLoading Cachefile datas...^[[m
[21:53:27] [Server thread/INFO]: ^[[0;36;22m[^[[0;36;1mMassiveCore 3.3.0^[[0;36;22m] ^[[0;33;1m^[[0;33;1mLoading Onlineplayer datas...^[[m
[21:53:27] [Server thread/INFO]: ^[[0;36;22m[^[[0;36;1mMassiveCore 3.3.0^[[0;36;22m] ^[[0;33;1m^[[0;33;1mLoading Registry datas...^[[m
[21:53:27] [Server thread/INFO]: ^[[0;36;22m[^[[0;36;1mMassiveCore 3.3.0^[[0;36;22m] ^[[0;33;1m^[[0;33;1mSaving Cachefile...^[[m
[21:53:27] [Server thread/INFO]: ^[[0;36;22m[^[[0;36;1mMassiveCore 3.3.0^[[0;36;22m] ^[[0;33;1m^[[0;33;1mSetup of IdUtil took ^[[0;35;1m14^[[0;33;1mms.^[[m
[21:53:28] [Server thread/ERROR]: Error occurred while enabling MassiveCore v3.3.0 (Is it up to date?)
java.lang.NoSuchFieldError: VERSION_1_1
        at com.massivecraft.massivecore.command.type.enumeration.TypeWorldType.getNamesInner(TypeWorldType.java:35) ~[?:?]
        at com.massivecraft.massivecore.command.type.enumeration.TypeWorldType.getNamesInner(TypeWorldType.java:8) ~[?:?]
        at com.massivecraft.massivecore.command.type.TypeAbstract.getNames(TypeAbstract.java:310) ~[?:?]
        at com.massivecraft.massivecore.command.type.TypeAbstractChoice.createOptionKeys(TypeAbstractChoice.java:359) ~[?:?]
        at com.massivecraft.massivecore.command.type.TypeAbstractChoice.createOptions(TypeAbstractChoice.java:339) ~[?:?]
        at com.massivecraft.massivecore.command.type.TypeAbstractChoice.setAll(TypeAbstractChoice.java:77) ~[?:?]
        at com.massivecraft.massivecore.command.type.TypeAbstractChoice.setAll(TypeAbstractChoice.java:84) ~[?:?]
        at com.massivecraft.massivecore.command.type.enumeration.TypeEnum.<init>(TypeEnum.java:25) ~[?:?]
        at com.massivecraft.massivecore.command.type.enumeration.TypeWorldType.<init>(TypeWorldType.java:18) ~[?:?]
        at com.massivecraft.massivecore.command.type.enumeration.TypeWorldType.<clinit>(TypeWorldType.java:14) ~[?:?]
        at com.massivecraft.massivecore.command.type.RegistryType.registerAll(RegistryType.java:318) ~[?:?]
        at com.massivecraft.massivecore.MassiveCore.onEnableInner(MassiveCore.java:256) ~[?:?]
        at com.massivecraft.massivecore.MassivePlugin.onEnable(MassivePlugin.java:84) ~[?:?]
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:263) ~[spigot-1.16.1.jar:git-Spigot-758abbe-3f0c333]
        at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:351) ~[spigot-1.16.1.jar:git-Spigot-758abbe-3f0c333]
        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:480) ~[spigot-1.16.1.jar:git-Spigot-758abbe-3f0c333]
        at org.bukkit.craftbukkit.v1_16_R1.CraftServer.enablePlugin(CraftServer.java:489) ~[spigot-1.16.1.jar:git-Spigot-758abbe-3f0c333]
        at org.bukkit.craftbukkit.v1_16_R1.CraftServer.enablePlugins(CraftServer.java:403) ~[spigot-1.16.1.jar:git-Spigot-758abbe-3f0c333]
        at net.minecraft.server.v1_16_R1.DedicatedServer.init(DedicatedServer.java:187) ~[spigot-1.16.1.jar:git-Spigot-758abbe-3f0c333]
        at net.minecraft.server.v1_16_R1.MinecraftServer.v(MinecraftServer.java:811) ~[spigot-1.16.1.jar:git-Spigot-758abbe-3f0c333]
        at net.minecraft.server.v1_16_R1.MinecraftServer.lambda$0(MinecraftServer.java:165) ~[spigot-1.16.1.jar:git-Spigot-758abbe-3f0c333]
        at java.lang.Thread.run(Thread.java:834) [?:?]

Plugins in use:
[21:53:26] [Server thread/INFO]: [WorldEdit] Loading WorldEdit v7.1.0;8e55131
[21:53:27] [Server thread/INFO]: [MassiveCore] Loading MassiveCore v3.3.0
[21:53:27] [Server thread/INFO]: [WorldGuard] Loading WorldGuard v7.0.2-SNAPSHOT;bf2593e
[21:53:27] [Server thread/INFO]: [Factions] Loading Factions v3.3.0

please help

**Comment by Puremin0rez** [comment]
The issue is that the "VERSION_1_1" world type was finally removed in 1.16.

I've gone ahead and simply removed it to make MassiveCore work again in 1.16.

I've attached the jar (zipped) to this comment. It should be compatible with the latest Spigot for 1.16.1.
[MassiveCore.zip](https://github.com/magnusulf/MassiveCore/files/4834028/MassiveCore.zip)

The only other breaking change in MassiveCore is the Zombie Pigman entity type, but I've left that alone since I didn't want to break backwards compatibility and the latest Spigot handles it via the compatibility layer.

I'd PR my changes, but the massive projects are massive and I can't validate everything works perfectly with it. I can at least confirm it works perfectly with CreativeGates.

Assuming @magnusulf is active, this should be pretty easy for him to resolve himself (based on how he wants to keep backwards compatibility or not)
